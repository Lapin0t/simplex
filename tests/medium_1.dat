50
30
0 -25 -74 -14 -82 0 0 27 14 0 0 0 0 89 0 0 0 -93 0 92 29 -16 0 -62 58 46 0 0 0 0 0 0 0 -20 -67 0 0 0 -68 0 0 0 0 0 -4 0 0 0 69 0
0 0 0 0 -68 21 0 -66 -54 0 0 0 -6 17 0 -20 0 0 86 0 0 0 17 66 0 98 0 0 0 0
-83 -50 -10 0 0 -75 -71 31 -59 0 78 38 -58 -93 0 0 0 -34 48 89 34 0 -29 -39 0 -65 0 0 47 0 0 0 0 46 -15 0 0 0 -66 0 -73 53 0 0 0 0 61 0 0 0
0 -13 0 0 0 4 0 2 74 -3 65 -2 -45 0 0 0 0 84 0 0 0 0 0 -78 0 -40 0 -64 0 0 0 15 -18 39 64 17 0 0 0 0 -37 0 0 0 0 0 0 -90 0 -95
0 92 0 0 0 0 22 10 -36 0 0 0 0 -57 0 0 4 67 -49 -77 0 0 29 -35 0 -82 0 0 0 -95 0 0 0 0 0 0 0 0 58 -21 -82 0 34 0 0 0 0 45 74 0
-60 -48 6 0 0 -38 0 0 0 -93 19 0 0 83 -8 0 0 81 -75 -70 20 -5 0 0 36 0 0 0 -41 0 0 0 0 0 -26 0 0 0 12 0 0 0 0 34 0 0 79 -81 0 83
-75 4 0 0 0 0 0 55 0 0 0 -83 0 0 0 0 0 0 0 97 69 0 0 10 0 23 0 29 8 62 51 0 0 64 0 -62 0 -87 0 -24 13 0 -35 0 0 0 0 0 -27 0
0 -3 0 10 0 0 0 0 -71 -83 77 0 0 -80 0 -24 0 0 0 0 0 36 -9 37 0 -13 0 13 -29 0 0 0 97 0 0 0 2 0 0 0 0 -90 0 -25 5 0 0 0 -37 27
0 -62 0 0 0 88 0 0 10 0 77 0 0 74 0 -9 -38 -78 52 0 -81 -24 0 89 4 0 0 0 0 0 0 0 -67 0 0 0 0 -83 0 41 0 57 0 0 0 0 0 0 0 0
0 0 0 0 -53 1 -34 0 -59 0 0 0 0 0 -83 0 -90 0 -98 0 0 0 0 33 -90 78 0 0 -99 0 -60 -63 0 0 -26 0 0 -64 0 0 0 0 -4 -81 -52 2 37 0 0 0
0 0 0 0 0 0 77 0 0 0 0 6 0 0 0 0 -6 0 0 -18 0 0 0 -70 0 -32 0 18 -19 0 17 -31 96 0 0 0 -53 0 73 0 0 0 -55 0 -24 52 -40 0 -21 0
21 0 39 41 61 0 -41 -53 0 0 0 60 0 0 12 0 0 62 5 34 0 -48 0 44 86 -72 -98 -11 98 0 0 0 0 -94 -40 0 85 0 0 51 0 0 -65 -13 69 -61 0 0 20 0
0 0 0 11 0 0 0 0 0 -14 0 0 0 -89 0 0 -66 21 0 75 0 26 0 0 0 0 0 -14 0 -93 0 -93 59 0 -69 89 -49 -54 0 0 0 0 -91 0 94 -57 0 0 0 0
26 0 -47 -99 0 0 0 94 0 13 0 0 71 13 0 -33 12 0 0 96 0 0 -28 -2 33 0 0 0 -22 47 0 0 0 0 0 74 52 0 0 11 0 -62 0 0 21 34 -48 -38 0 97
0 0 79 33 0 0 0 0 0 -41 0 0 35 -93 0 0 0 0 0 0 30 0 18 0 0 0 0 0 0 -88 0 0 0 -99 0 0 0 -57 0 0 99 -2 0 18 0 0 0 -55 -48 49
0 0 -68 0 0 0 57 -66 0 0 0 0 0 -100 0 -79 -4 0 85 92 0 0 0 0 0 0 0 0 0 9 0 0 -21 0 0 -92 0 4 0 0 82 -64 0 -36 0 0 49 0 0 0
0 0 81 0 56 -51 -5 0 0 0 0 0 -18 0 19 0 0 0 0 -6 0 0 0 0 -22 0 -73 0 0 -36 0 24 0 0 0 0 -94 0 0 96 -81 96 0 0 -37 0 -40 0 -62 -67
89 9 0 0 0 -20 0 -8 0 -66 0 0 0 0 -87 39 90 0 -94 -98 30 -59 0 0 0 0 0 0 -9 67 0 0 0 0 -17 0 66 0 -75 -48 0 -33 0 0 -92 0 0 0 0 23
58 -58 78 -32 0 40 0 0 0 0 0 0 0 -24 47 20 -45 -13 84 -27 0 0 3 0 0 0 0 0 0 0 0 -97 0 0 0 33 0 -81 -78 0 96 0 0 0 47 9 0 3 44 0
-5 0 0 31 0 -34 0 0 0 0 -32 0 0 -48 -35 0 -57 0 0 0 0 0 0 0 0 57 48 -47 51 0 -47 0 0 -89 0 0 0 0 73 -44 0 0 0 0 0 0 59 99 0 -26
0 0 -70 0 0 4 0 0 -20 0 0 -82 0 97 0 0 -86 0 0 -77 65 -30 -93 0 0 0 0 0 0 0 -88 0 0 0 0 -67 44 0 0 -81 -58 0 -64 13 0 0 11 32 11 0
0 54 -54 0 0 0 60 0 -78 0 -22 0 0 0 0 98 -33 0 0 -17 -18 0 0 0 -62 -43 -82 0 0 -73 0 -25 67 18 0 -6 0 0 -74 -89 0 -73 -74 0 -92 17 0 -32 0 0
0 0 0 0 -54 0 45 93 0 0 0 0 0 0 -48 3 85 0 -16 -85 0 0 0 0 0 11 -48 0 5 67 38 0 0 92 32 -90 31 0 51 -81 0 0 18 0 0 26 9 -17 -5 0
30 0 0 0 -94 0 0 -56 0 0 0 61 0 -40 0 -71 2 60 -8 0 0 0 0 0 93 52 0 0 75 -15 0 -81 0 0 10 0 0 -76 43 0 0 0 0 0 0 94 75 0 50 94
0 -38 84 97 0 0 -29 -42 0 0 -99 0 -90 -87 0 -4 0 30 0 0 0 0 -53 0 -67 32 0 59 0 0 0 -40 0 0 0 88 -32 0 0 0 0 0 0 74 0 0 0 68 40 0
0 0 50 0 0 15 0 0 -82 -60 0 0 0 0 0 64 0 0 0 21 0 0 0 -80 -20 95 0 0 0 76 -9 79 0 55 -17 0 0 0 83 0 0 0 0 0 0 0 10 0 -9 98
0 0 0 0 0 0 35 0 0 0 0 0 0 -70 30 0 0 0 0 0 0 0 -87 0 0 0 -20 0 0 42 41 0 0 0 0 -37 0 0 -39 -71 0 -30 0 0 55 33 51 0 -97 0
-6 -29 97 0 0 0 0 -80 0 0 -13 0 0 0 0 70 0 0 27 0 57 0 0 0 0 0 0 -76 -86 -67 0 -93 93 -88 0 72 50 -74 47 0 0 -2 95 -3 0 0 83 0 0 0
0 11 -11 -66 18 0 0 0 0 0 -42 0 77 0 83 -85 0 -83 0 -49 66 0 25 69 17 67 0 -51 -4 0 0 -16 0 0 0 0 0 -89 0 -64 90 0 0 0 0 0 -50 -51 65 0
0 0 75 0 0 50 0 23 48 20 0 0 -68 -26 0 0 -2 0 0 0 0 45 2 0 99 17 -7 0 -7 0 -47 0 0 0 -29 18 0 0 0 0 0 0 0 0 -50 0 0 10 30 0
0 0 0 0 0 -50 -79 25 0 58 0 87 -26 0 0 0 91 13 0 0 0 0 0 0 58 0 0 0 0 67 45 0 0 0 76 0 0 0 0 0 0 -99 0 0 0 -62 0 0 0 0
0 0 0 -2 0 -74 0 61 0 -63 99 56 0 0 0 0 0 88 0 0 0 0 37 -12 82 0 32 0 47 0 73 7 0 -61 0 27 31 0 0 -42 0 10 0 0 0 0 0 -36 0 0
