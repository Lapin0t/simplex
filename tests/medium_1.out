INFO: maximize:
INFO:          -25x_1 -74x_2 -14x_3 -82x_4               +27x_7 +14x_8                                 +89x_13                         -93x_17         +92x_19 +29x_20 -16x_21         -62x_23 +58x_24 +46x_25                                                         -20x_33 -67x_34                         -68x_38                                          -4x_44                         +69x_48        
INFO: subject to:
INFO:   -83x_0 -50x_1 -10x_2               -75x_5 -71x_6 +31x_7 -59x_8        +78x_10 +38x_11 -58x_12  -93x_13                         -34x_17 +48x_18 +89x_19 +34x_20         -29x_22 -39x_23         -65x_25                 +47x_28                                 +46x_33 -15x_34                         -66x_38         -73x_40 +53x_41                                 +61x_46                         <= 0
INFO:          -13x_1                       +4x_5         +2x_7 +74x_8  -3x_9 +65x_10  -2x_11 -45x_12                                  +84x_17                                         -78x_23         -40x_25         -64x_27                         +15x_31 -18x_32 +39x_33 +64x_34 +17x_35                                 -37x_40                                                 -90x_47         -95x_49 <= 0
INFO:          +92x_1                             +22x_6 +10x_7 -36x_8                                 -57x_13                  +4x_16 +67x_17 -49x_18 -77x_19                 +29x_22 -35x_23         -82x_25                         -95x_29                                                                 +58x_38 -21x_39 -82x_40         +34x_42                                 +45x_47 +74x_48         <= 0
INFO:   -60x_0 -48x_1  +6x_2               -38x_5                      -93x_9 +19x_10                  +83x_13  -8x_14                 +81x_17 -75x_18 -70x_19 +20x_20  -5x_21                 +36x_24                         -41x_28                                         -26x_34                         +12x_38                                 +34x_43                 +79x_46 -81x_47         +83x_49 <= 0
INFO:   -75x_0  +4x_1                                    +55x_7                       -83x_11                                                          +97x_19 +69x_20                 +10x_23         +23x_25         +29x_27  +8x_28 +62x_29 +51x_30                 +64x_33         -62x_35         -87x_37         -24x_39 +13x_40         -35x_42                                         -27x_48         <= -68
INFO:           -3x_1        +10x_3                             -71x_8 -83x_9 +77x_10                  -80x_13         -24x_15                                         +36x_21  -9x_22 +37x_23         -13x_25         +13x_27 -29x_28                         +97x_32                          +2x_36                                 -90x_41         -25x_43  +5x_44                         -37x_48 +27x_49 <= 21
INFO:          -62x_1                      +88x_5               +10x_8        +77x_10                  +74x_13          -9x_15 -38x_16 -78x_17 +52x_18         -81x_20 -24x_21         +89x_23  +4x_24                                                         -67x_32                                 -83x_37         +41x_39         +57x_41                                                                 <= 0
INFO:                               -53x_4   +x_5 -34x_6        -59x_8                                         -83x_14         -90x_16         -98x_18                                 +33x_23 -90x_24 +78x_25                 -99x_28         -60x_30 -63x_31                 -26x_34                 -64x_37                                  -4x_42 -81x_43 -52x_44  +2x_45 +37x_46                         <= -66
INFO:                                             +77x_6                               +6x_11                                   -6x_16                 -18x_19                         -70x_23         -32x_25         +18x_27 -19x_28         +17x_30 -31x_31 +96x_32                         -53x_36         +73x_38                         -55x_42         -24x_44 +52x_45 -40x_46         -21x_48         <= -54
INFO:   +21x_0        +39x_2 +41x_3 +61x_4        -41x_6 -53x_7                       +60x_11                  +12x_14                 +62x_17  +5x_18 +34x_19         -48x_21         +44x_23 +86x_24 -72x_25 -98x_26 -11x_27 +98x_28                                 -94x_33 -40x_34         +85x_36                 +51x_39                 -65x_42 -13x_43 +69x_44 -61x_45                 +20x_48         <= 0
INFO:                        +11x_3                                    -14x_9                          -89x_13                 -66x_16 +21x_17         +75x_19         +26x_21                                         -14x_27         -93x_29         -93x_31 +59x_32         -69x_34 +89x_35 -49x_36 -54x_37                                 -91x_42         +94x_44 -57x_45                                 <= 0
INFO:   +26x_0        -47x_2 -99x_3                      +94x_7        +13x_9                 +71x_12  +13x_13         -33x_15 +12x_16                 +96x_19                 -28x_22  -2x_23 +33x_24                         -22x_28 +47x_29                                         +74x_35 +52x_36                 +11x_39         -62x_41                 +21x_44 +34x_45 -48x_46 -38x_47         +97x_49 <= 0
INFO:                 +79x_2 +33x_3                                    -41x_9                 +35x_12  -93x_13                                                 +30x_20         +18x_22                                                 -88x_29                         -99x_33                         -57x_37                 +99x_40  -2x_41         +18x_43                         -55x_47 -48x_48 +49x_49 <= -6
INFO:                 -68x_2                      +57x_6 -66x_7                                       -100x_13         -79x_15  -4x_16         +85x_18 +92x_19                                                                          +9x_29                 -21x_32                 -92x_35          +4x_37                 +82x_40 -64x_41         -36x_43                 +49x_46                         <= 17
INFO:                 +81x_2        +56x_4 -51x_5  -5x_6                                      -18x_12          +19x_14                                  -6x_19                                 -22x_24         -73x_26                 -36x_29         +24x_31                                 -94x_36                 +96x_39 -81x_40 +96x_41                 -37x_44         -40x_46         -62x_48 -67x_49 <= 0
INFO:   +89x_0  +9x_1                      -20x_5         -8x_7        -66x_9                                  -87x_14 +39x_15 +90x_16         -94x_18 -98x_19 +30x_20 -59x_21                                                  -9x_28 +67x_29                                 -17x_34         +66x_36         -75x_38 -48x_39         -33x_41                 -92x_44                                 +23x_49 <= -20
INFO:   +58x_0 -58x_1 +78x_2 -32x_3        +40x_5                                                      -24x_13 +47x_14 +20x_15 -45x_16 -13x_17 +84x_18 -27x_19                  +3x_22                                                                 -97x_31                         +33x_35         -81x_37 -78x_38         +96x_40                         +47x_44  +9x_45          +3x_47 +44x_48         <= 0
INFO:    -5x_0               +31x_3        -34x_5                             -32x_10                  -48x_13 -35x_14         -57x_16                                                                 +57x_25 +48x_26 -47x_27 +51x_28         -47x_30                 -89x_33                                 +73x_38 -44x_39                                                 +59x_46 +99x_47         -26x_49 <= 0
INFO:                 -70x_2                +4x_5               -20x_8                -82x_11          +97x_13                 -86x_16                 -77x_19 +65x_20 -30x_21 -93x_22                                                         -88x_30                                 -67x_35 +44x_36                 -81x_39 -58x_40         -64x_42 +13x_43                 +11x_46 +32x_47 +11x_48         <= 86
INFO:          +54x_1 -54x_2                      +60x_6        -78x_8        -22x_10                                  +98x_15 -33x_16                 -17x_19 -18x_20                         -62x_24 -43x_25 -82x_26                 -73x_29         -25x_31 +67x_32 +18x_33          -6x_35                 -74x_38 -89x_39         -73x_41 -74x_42         -92x_44 +17x_45         -32x_47                 <= 0
INFO:                               -54x_4        +45x_6 +93x_7                                                -48x_14  +3x_15 +85x_16         -16x_18 -85x_19                                         +11x_25 -48x_26          +5x_28 +67x_29 +38x_30                 +92x_33 +32x_34 -90x_35 +31x_36         +51x_38 -81x_39                 +18x_42                 +26x_45  +9x_46 -17x_47  -5x_48         <= 0
INFO:   +30x_0                      -94x_4               -56x_7                       +61x_11          -40x_13         -71x_15  +2x_16 +60x_17  -8x_18                                         +93x_24 +52x_25                 +75x_28 -15x_29         -81x_31                 +10x_34                 -76x_37 +43x_38                                                 +94x_45 +75x_46         +50x_48 +94x_49 <= 0
INFO:          -38x_1 +84x_2 +97x_3               -29x_6 -42x_7               -99x_10         -90x_12  -87x_13          -4x_15         +30x_17                                 -53x_22         -67x_24 +32x_25         +59x_27                         -40x_31                         +88x_35 -32x_36                                                 +74x_43                         +68x_47 +40x_48         <= 17
INFO:                 +50x_2               +15x_5               -82x_8 -60x_9                                          +64x_15                         +21x_19                         -80x_23 -20x_24 +95x_25                         +76x_29  -9x_30 +79x_31         +55x_33 -17x_34                         +83x_38                                                         +10x_46          -9x_48 +98x_49 <= 66
INFO:                                             +35x_6                                               -70x_13 +30x_14                                                         -87x_22                         -20x_26                 +42x_29 +41x_30                                 -37x_35                 -39x_38 -71x_39         -30x_41                 +55x_44 +33x_45 +51x_46         -97x_48         <= 0
INFO:    -6x_0 -29x_1 +97x_2                             -80x_7               -13x_10                                  +70x_15                 +27x_18         +57x_20                                                 -76x_27 -86x_28 -67x_29         -93x_31 +93x_32 -88x_33         +72x_35 +50x_36 -74x_37 +47x_38                  -2x_41 +95x_42  -3x_43                 +83x_46                         <= 98
INFO:          +11x_1 -11x_2 -66x_3 +18x_4                                    -42x_10         +77x_12          +83x_14 -85x_15         -83x_17         -49x_19 +66x_20         +25x_22 +69x_23 +17x_24 +67x_25         -51x_27  -4x_28                 -16x_31                                         -89x_37         -64x_39 +90x_40                                         -50x_46 -51x_47 +65x_48         <= 0
INFO:                 +75x_2               +50x_5        +23x_7 +48x_8 +20x_9                 -68x_12  -26x_13                  -2x_16                                 +45x_21  +2x_22         +99x_24 +17x_25  -7x_26          -7x_28         -47x_30                         -29x_34 +18x_35                                                                 -50x_44                 +10x_47 +30x_48         <= 0
INFO:                                      -50x_5 -79x_6 +25x_7        +58x_9         +87x_11 -26x_12                          +91x_16 +13x_17                                                 +58x_24                                 +67x_29 +45x_30                         +76x_34                                                 -99x_41                         -62x_45                                 <= 0
INFO:                         -2x_3        -74x_5        +61x_7        -63x_9 +99x_10 +56x_11                                          +88x_17                                 +37x_22 -12x_23 +82x_24         +32x_26         +47x_28         +73x_30  +7x_31         -61x_33         +27x_35 +31x_36                 -42x_39         +10x_41                                         -36x_47                 <= 0
INFO:   x_0, x_1, x_2, x_3, x_4, x_5, x_6, x_7, x_8, x_9, x_10, x_11, x_12, x_13, x_14, x_15, x_16, x_17, x_18, x_19, x_20, x_21, x_22, x_23, x_24, x_25, x_26, x_27, x_28, x_29, x_30, x_31, x_32, x_33, x_34, x_35, x_36, x_37, x_38, x_39, x_40, x_41, x_42, x_43, x_44, x_45, x_46, x_47, x_48, x_49, x_50, x_51, x_52, x_53, x_54, x_55, x_56, x_57, x_58, x_59, x_60, x_61, x_62, x_63, x_64, x_65, x_66, x_67, x_68, x_69, x_70, x_71, x_72, x_73, x_74, x_75, x_76, x_77, x_78, x_79 >= 0
INFO: number of variables: 50
INFO: number of constraints: 30
INFO: PHASE 1 (5 bad constraints)
INFO: PHASE 2 (initial base: [50 51 25 48 62 55 56 14 22  3 60 12 37 63 41 13 66 54 27 69 70 71 64 73 74
 75 65  5  9 79])
INFO: number of pivot steps (both phases): 26
INFO: rule used: greedy
UNBOUNDED
